signalbot/command.py

```diff
@@ -1,10 +1,30 @@
 import functools
+import re
 from abc import ABC, abstractmethod
+from typing import Pattern
 
 from .message import Message
 from .context import Context
 
 
+def regex_triggered(*by: str | Pattern[str]):
```

Comment:
Importing from typing is deprecated, https://docs.python.org/3/library/typing.html#typing.Pattern , you should be able to import from re
```suggestion
def regex_triggered(*by: str | re.Pattern[str]):
```